examples/qml_minimal/CMakeLists.txt

```diff
@@ -78,20 +82,37 @@ add_library(${APP_NAME}_lib INTERFACE)
 # overwriting another's files.
 target_include_directories(${APP_NAME}_lib INTERFACE "${CXXQT_EXPORT_DIR}/${CRATE}")
 
-target_link_libraries(${APP_NAME}_lib INTERFACE
-    # WHOLE_ARCHIVE is needed for the generated QML plugin to register on startup,
-    # otherwise the linker will discard the static variables that initialize it.
-    "$<LINK_LIBRARY:WHOLE_ARCHIVE,${CRATE}-static>"
-    Qt::Core
-    Qt::Gui
-    Qt::Qml
-    Qt::QuickControls2
-)
+if(BUILD_WASM)
+    target_link_libraries(${APP_NAME}_lib INTERFACE
+        Qt::Core
+        Qt::Gui
+        Qt::Qml
+        Qt::QuickControls2
+    )
+    
+    target_link_options(${APP_NAME}_lib INTERFACE 
```

Comment:
vscode seems to think there is an extra space at the end of this line and potentially in the line before too